ui-grid/ui-grid.d.ts

```diff
@@ -2823,12 +2823,12 @@ declare module uiGrid {
              * returns all selected rows as gridRows
              * @returns {Array<IGridRow>} The selected rows
              */
-            getSelectedGridRows(): Array<IGridRow>;
+            getSelectedGridRows(): Array<any>;
```

Comment:
Actually, @baltie , is this one supposed to be `any`? It looks like this one is supposed to return the gridRow objects rather than the entities.

https://github.com/angular-ui/ui-grid/blob/314f1ed/src/features/selection/js/selection.js#L292

as opposed to 

https://github.com/angular-ui/ui-grid/blob/314f1ed/src/features/selection/js/selection.js#L281